common/script/roleInfo.js

```diff
@@ -3291,15 +3272,15 @@
   "image": {
     "name": "image",
     "fragID": "image",
-    "parentRoles": [],
+    "parentRoles": [
```

Comment:
The changes to image/img stem from image becoming the default #2076

Reply:
But this is also something we want to fix. Cf. also https://github.com/w3c/aria/issues/2073